include/ck/tensor_operation/gpu/block/reduction_functions_blockwise.hpp

```diff
@@ -145,24 +168,24 @@ struct PartitionedBlockwiseReductionWithIndex
                 index_t offset2 = block_buf_desc_m_k.CalculateOffset(thread_cluster_idx +
                                                                      make_tuple(0, indOffset));
 
-                AccDataType opData1      = type_convert<AccDataType>(block_val_buffer[offset1]);
-                AccDataType opData2      = type_convert<AccDataType>(block_val_buffer[offset2]);
-                IndexDataType currIndex1 = block_idx_buffer[offset1];
-                IndexDataType currIndex2 = block_idx_buffer[offset2];
+                AccDataType opData1      = work_val_buffer[offset1];
```

Comment:
same here, type_convert

Reply:
Will add static_assert to ensure the data type of work_val_buffer be consistent with the AccDataType

Reply:
Please add static_assert in this PR

Reply:
Please add static_assert in this PR

Reply:
This is done in [commit 8648](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/152/commits/ded71f1f4c1135ffb04923eda103a0adfb3c8648)